src/components/login/login.test.js

```diff
@@ -85,18 +86,18 @@ describe('Login', () => {
   });
 
   describe('History management', () => {
-    it('calls this.props.history.replace(\'/main/dashboard\')', () => {
+    it(`calls this.props.history.replace('${routes.main}${routes.dashboard.url}')`, () => {
```

Comment:
Please update the `it` blocks with descriptions that are more readable (as you were already changing them anyway). We want to avoid exactly the situation that you were in here, where when we change some implementation, the describe blocks have to be updated too. Instead, by having a general readable description of what we are expecting, there will be no need to always update the blocks. So one possible example could be, instead of 
`calls this.props.history.replace('${routes.main}${routes.dashboard.url}')` -> `redirects to dashboard`